.github/workflows/build-with-bal-test-native.yml

```diff
@@ -15,3 +15,5 @@ jobs:
     if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
     uses: ballerina-platform/ballerina-standard-library/.github/workflows/build-with-bal-test-graalvm-connector-template.yml@main
     secrets: inherit
+    with:
+      additional-build-flags: "-x :googleapis.gcalendar-examples:build -Pgroups=mock"
```

Comment:
Since there are no ENV configured in environment, IMO we don't need no specify in `-Pgroups=mock`.

Reply:
Since there are test cases that only runs with "live" group, we have to specify the test group here